tools-ui/src/main/java/com/hedera/hashgraph/client/ui/Controller.java

```diff
@@ -837,4 +834,23 @@ public void loadPubKeys() {
 	public Map<String, Path> getPubFiles() {
 		return keyStructureUtility.getPubFiles();
 	}
+
+	public JsonObject getAccountsList() {
+		JsonObject object = new JsonObject();
+		try {
+			object = (new File(ACCOUNTS_MAP_FILE).exists()) ? readJsonObject(ACCOUNTS_MAP_FILE) : new JsonObject();
+		} catch (HederaClientException e) {
+			logger.error(e.getMessage());
+		}
+		return object;
+	}
+
+	public void setLastBrowsedDirectory(File file) {
+		if (file.isDirectory()) {
+			properties.setLastBrowsedDirectory(file);
+		}
+		if (file.isFile()) {
```

Comment:
nit: an `else` in front of `if` on this line may save another comparison.

Reply:
good catch